tests/unittest/test_loss.py

```diff
@@ -32,3 +33,33 @@ def testTemporalActivationRegularizationLoss():
     print(tar)
     tar(*[mx.nd.arange(1000).reshape(10, 10, 10),
           mx.nd.arange(1000).reshape(10, 10, 10)])
+    
+def testMaskedSoftmaxCrossEntropyLoss():
+    loss_fn = nlp.loss.SoftmaxCEMaskedLoss()
+    pred = mx.nd.array([[[0,0,10],[10,0,0]]]) #N,T,C 1,2,3
+    label = mx.nd.array([[2,2]])
+    valid_length = mx.nd.array([1,]) 
+    loss = loss_fn(pred, label, valid_length)
+    assert loss < 0.1, "1st timestep prediction is correct"
+    valid_length = mx.nd.array([2,]) 
+    loss = loss_fn(pred, label, valid_length)
+    assert loss > 1, "2nd timestep prediction is wrong, loss goes up"
```

Comment:
Likewise here, the message is printed when the assertion fails. e.g., instead write "2nd timestep prediction was wrong, but loss did not go up"